src/main/kotlin/test/ktortemplate/core/exception/ExceptionHandler.kt

```diff
@@ -0,0 +1,39 @@
+package test.ktortemplate.core.exception
+
+import io.ktor.application.*
+import io.ktor.features.StatusPages.*
+import io.ktor.http.*
+import io.ktor.response.*
+
+private data class ErrorData (
```

Comment:
This is not consistent with what we have on the documentation, here: https://github.com/waterdog-oss/ktor-template/wiki/Error-handling